legend-engine-xt-sql-postgres-server/src/main/java/org/finos/legend/engine/postgres/handler/legend/LegendResultSet.java

```diff
@@ -61,6 +61,14 @@ public Object getObject(int i) throws Exception
     {
         LegendColumn legendColumn = columns.get(i - 1);
         String value = currentRow.get(i - 1);
+        if (value.equals("null"))
```

Comment:
As discussed, this feels unexpected to be having raw strings at this point in the flow.  Is that correct?